.github/workflows/publish-crates.yml

```diff
@@ -1,12 +1,9 @@
 name: Publish Crates
 
 on:
-  workflow_dispatch:
-    inputs:
-      version:
-        description: 'Version to publish (without v prefix)'
-        required: true
-        type: string
+  workflow_dispatch: {}
```

Comment:
```suggestion
  workflow_dispatch:
```

Reply:
I actually added that `{}` because it seemed so odd to have that `workflow_dispatch:` hanging there...